fhir-install/docker/Dockerfile-fhirdb2

```diff
@@ -0,0 +1,21 @@
+#
+# (C) Copyright IBM Corp. 2019
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+FROM ibmcom/db2
+
+# Create a non-admin user for the FHIR server
+RUN groupadd -g 1002 fhir
+RUN  useradd -u 1002 -g fhir -M -d /database/config/fhiruser fhiruser 
+RUN echo "change-password" | passwd --stdin fhiruser
```

Comment:
please add a comment, and identify as experimental

Reply:
good point, will add

Reply:
done